.travis.yml

```diff
@@ -57,6 +64,24 @@ before_install:
         $DOCKER)
 
       SCRIPT_RUN_PREFIX="docker exec --tty $containerid"
+    elif [[ "$RUNMODE" == "singularity" ]] ; then
+      # prepare Singularity
```

Comment:
How much time does this build step take?  I don't want it to too long.

Reply:
If singularity has a Ubuntu package we probably should use it to save install time?

Reply:
There is no deb package for Ubuntu so far. It takes so much time because it builds from scratch, including an Ubuntu image. In my opinion, we could prepare a cache of the base image to reduce the build time.

Reply:
That is better than building it on the fly in this travis file.